commit dc4b49ce0642d559806e399ba0bfb340be7970b7
Author: WANG Xuerui <git@xen0n.name>
Date:   Fri Jun 26 16:14:23 2020 +0800

    [stage2] pacman: fix patch

diff --git a/PKGBUILD b/PKGBUILD
index 98f2a5e..9f6b843 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -33,7 +33,7 @@ sha256sums=('1930c407265fd039cb3a8e6edc82f69e122aa9239d216d9d57b9d1b9315af312'
             'e481a161bba76729cd434c97e0b319ddfcb1d93b2e4890d72b4e8a32982531d9'
             '667ba659f85e3740fda9808e4751a44a63e0484072594d961b87e474c607b79c'
             '3353f363088c73f1f86a890547c0f87c7473e5caf43bbbc768c2e9a7397f2aa2'
-            '8c100b64450f5a19a16325dd05c143d49395bdeb96bd957f863cde4b95d3cb86')
+            'd9a3a9a565bfdfe9c05fb891c415f9fe113581e0a3c5caa32534ee4fba152863')
 
 prepare() {
   cd "$pkgname-$pkgver"
@@ -46,7 +46,8 @@ build() {
   cd "$pkgname-$pkgver"
 
   ./configure --prefix=/usr --sysconfdir=/etc \
-    --localstatedir=/var --enable-doc \
+    --host=@CHOST@ --build=@BUILDHOST@ \
+    --localstatedir=/var --disable-doc \
     --with-scriptlet-shell=/usr/bin/bash \
     --with-ldconfig=/usr/bin/ldconfig
   make V=1
diff --git a/makepkg.conf b/makepkg.conf
index cecfd9d..094966a 100644
--- a/makepkg.conf
+++ b/makepkg.conf
@@ -32,13 +32,13 @@ VCSCLIENTS=('bzr::bzr'
 # ARCHITECTURE, COMPILE FLAGS
 #########################################################################
 #
-CARCH="x86_64"
-CHOST="x86_64-pc-linux-gnu"
+CARCH="mips64el"
+CHOST="mips64el-unknown-linux-gnuabi64"
 
 #-- Compiler and Linker Flags
 CPPFLAGS="-D_FORTIFY_SOURCE=2"
-CFLAGS="-march=x86-64 -mtune=generic -O2 -pipe -fno-plt"
-CXXFLAGS="-march=x86-64 -mtune=generic -O2 -pipe -fno-plt"
+CFLAGS="-march=mips64r2 -mabi=64 -O2 -pipe -fstack-protector-strong -fno-plt"
+CXXFLAGS="-march=mips64r2 -mabi=64 -O2 -pipe -fstack-protector-strong -fno-plt"
 LDFLAGS="-Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now"
 #-- Make Flags: change this for DistCC/SMP systems
 #MAKEFLAGS="-j2"
