commit 6f5a5f5830ab61fa2c5a77ed0f4ebaabd39e0d7c
Author: WANG Xuerui <git@xen0n.name>
Date:   Thu Jun 25 21:50:23 2020 +0800

    [stage2] curl: fix patch

diff --git a/PKGBUILD b/PKGBUILD
index 6287b85..3913305 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -11,7 +11,7 @@ pkgdesc="An URL retrieval utility and library"
 arch=('x86_64')
 url="https://curl.haxx.se"
 license=('MIT')
-depends=('ca-certificates' 'krb5' 'libssh2' 'openssl' 'zlib' 'libpsl' 'libnghttp2'
+depends=('ca-certificates' 'libssh2' 'openssl' 'zlib' 'libpsl' 'libnghttp2'
          'libidn2' 'libidn2.so')
 provides=('libcurl.so')
 source=("https://curl.haxx.se/download/$pkgname-$pkgver.tar.gz"{,.asc})
@@ -23,6 +23,7 @@ build() {
   cd "$pkgname-$pkgver"
 
   ./configure \
+      --host=@CHOST@ --build=@BUILDHOST@ \
       --prefix=/usr \
       --mandir=/usr/share/man \
       --disable-ldap \
