diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2019-01-29 02:04:14.211582438 +0300
+++ b/PKGBUILD	2019-01-29 02:07:45.823290530 +0300
@@ -11,7 +11,7 @@ pkgdesc="Freedesktop.org message bus sys
 url="https://wiki.freedesktop.org/www/Software/dbus/"
 arch=(x86_64)
 license=(GPL custom)
-depends=(libsystemd expat audit)
+depends=(libsystemd expat glib2)
 makedepends=(systemd xmlto docbook-xsl python yelp-tools doxygen git autoconf-archive graphviz)
 _commit=d4f8423bbff9b3c5fca2d8009c28d1cff4652788  # tags/dbus-1.12.12^0
 source=("git+https://gitlab.freedesktop.org/dbus/dbus.git#commit=$_commit")
@@ -37,6 +37,8 @@ prepare() {
 build() {
   cd dbus
   ./configure \
+    -host=@CHOST@ --build=@BUILDHOST@ \
+    --disable-silent-rules --disable-xml-docs \
     --prefix=/usr \
     --sysconfdir=/etc \
     --localstatedir=/var \
