diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2019-01-23 17:36:16.827394862 +0300
+++ b/PKGBUILD	2019-01-23 17:49:59.605906163 +0300
@@ -25,7 +25,7 @@ build() {
   export CFLAGS=${CFLAGS/-fno-plt} # fix make check with i686 
 
   CFLAGS+=" -g"  # required for test-suite success
-  ./configure --prefix=/usr --program-prefix="eu-" --enable-deterministic-archives
+  ./configure --host=@CHOST@ --build=@BUILDHOST@ --prefix=/usr --program-prefix="eu-" --enable-deterministic-archives
 
   make
 }
