commit 50abf9a865b92dd535a7bea33e83c60675db8ab2
Author: WANG Xuerui <git@xen0n.name>
Date:   Fri Jun 26 12:56:41 2020 +0800

    [stage2] findutils: fix patch

diff --git a/PKGBUILD b/PKGBUILD
index 4e2ac8b..0c10bca 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -21,7 +21,7 @@ build() {
   # which is a secure version of locate.
   sed -e '/^SUBDIRS/s/locate//' -e 's/frcode locate updatedb//' -i Makefile.in
 
-  ./configure --prefix=/usr
+  ./configure --host=@CHOST@ --build=@BUILDHOST@ --prefix=/usr
   # don't build locate, but the docs want a file in there.
   make -C locate dblocation.texi
   make
