diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-11-08 14:53:25.000000000 +0300
+++ b/PKGBUILD	2019-02-05 14:29:44.946293593 +0300
@@ -21,6 +21,7 @@ prepare() {
   cd "${srcdir}/${pkgname}-${pkgver}"
   patch -p1 -i "$srcdir"/gnulib-fflush.patch
   patch -p1 -i "$srcdir"/gnulib-makedev.patch
+  autoreconf --install
 }
 
 build() {
@@ -30,7 +31,7 @@ build() {
   # which is a secure version of locate.
   sed -i '/^SUBDIRS/s/locate//' Makefile.in
 
-  ./configure --prefix=/usr
+  ./configure --host=@CHOST@ --build=@BUILDHOST@ --prefix=/usr
   # don't build locate, but the docs want a file in there.
   make -C locate dblocation.texi
   make
