diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-11-08 15:04:36.000000000 +0300
+++ b/PKGBUILD	2019-02-05 16:12:53.714093485 +0300
@@ -20,12 +20,13 @@ md5sums=('2882e3179748cc9f9c23ec593d6adc
 prepare() {
   cd "$pkgname-$pkgver"
   patch -p1 -i "$srcdir"/flex-pie.patch
+  sed -i '/doc \\/d' Makefile.am
   autoreconf
 }
 
 build() {
   cd "$pkgname-$pkgver"
-  ./configure --prefix=/usr
+  ./configure --prefix=/usr --host=@CHOST@ --build=@BUILDHOST@ --disable-bootstrap
   make
 }
 
