commit 4dbb24bec5a0781033348033fd10884e6f4e169e
Author: WANG Xuerui <git@xen0n.name>
Date:   Fri Jun 26 16:23:36 2020 +0800

    [stage2] gnutls: disable guile

diff --git a/PKGBUILD b/PKGBUILD
index f10bf88..f1a3406 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -22,12 +22,16 @@ validpgpkeys+=('462225C3B46F34879FC8496CD605848ED7E69871') # "Daiki Ueno <ueno@u
 
 build() {
   cd ${pkgname}-${pkgver}
+  export PKG_CONFIG_LIBDIR="@SYSROOT@/usr/lib/pkgconfig"
+  export PKG_CONFIG_SYSROOT_DIR="@SYSROOT@"
   ./configure --prefix=/usr \
+	--host=@CHOST@ --build=@BUILDHOST@ \
+	--with-libunistring-prefix=@SYSROOT@/usr \
 	--with-zlib \
 	--disable-static \
 	--with-idn \
 	--enable-openssl-compatibility \
-	--enable-guile --with-guile-site-dir=no \
+	--disable-guile --with-guile-site-dir=no \
 	--with-default-trust-store-pkcs11="pkcs11:model=p11-kit-trust;manufacturer=PKCS%2311%20Kit"
   sed -i -e 's/ -shared / -Wl,-O1,--as-needed\0/g' libtool
   make
