commit ca770fb7cd2a90d00ab5cdfb3ba55b8a2ff29868
Author: WANG Xuerui <git@xen0n.name>
Date:   Fri Jun 26 16:17:06 2020 +0800

    [stage2] gpgme: fix patch and build

diff --git a/PKGBUILD b/PKGBUILD
index 4c11d99..6bf803f 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -3,7 +3,7 @@
 # Contributor: Sarah Hay <sarah@archlinux.org>
 
 pkgbase=gpgme
-pkgname=(gpgme qgpgme python-gpgme)
+pkgname=(gpgme)
 pkgver=1.13.1
 pkgrel=7
 _python_ver=3.8
@@ -22,15 +22,17 @@ build() {
 
   export PYTHONHASHSEED=0
   ./configure \
+    --host=@CHOST@ --build=@BUILDHOST@ \
     --prefix=/usr \
     --disable-fd-passing \
     --disable-static \
+    --disable-gpg-test \
     --disable-gpgsm-test
   make
 
   # ensure reproducibility of .pyc files
-  touch -d @$SOURCE_DATE_EPOCH lang/python/version.py
-  touch -d @$SOURCE_DATE_EPOCH lang/python/python${_python_ver}-gpg/lib.linux-x86_64-${_python_ver}/gpg/gpgme.py
+  #touch -d @$SOURCE_DATE_EPOCH lang/python/version.py
+  #touch -d @$SOURCE_DATE_EPOCH lang/python/python${_python_ver}-gpg/lib.linux-x86_64-${_python_ver}/gpg/gpgme.py
   
   # .pyc files will be created if check() is used - generate them here
   
@@ -59,9 +61,9 @@ package_gpgme() {
   make DESTDIR="${pkgdir}" install
 
   # split qgpgme
-  rm -r "${pkgdir}"/usr/include/{qgpgme,QGpgME}/
-  rm -r "${pkgdir}"/usr/lib/{cmake/QGpgme/,libqgpgme.*}
-  rm -r "${pkgdir}"/usr/lib/python*
+  rm -r "${pkgdir}"/usr/include/{qgpgme,QGpgME}/ || true
+  rm -r "${pkgdir}"/usr/lib/{cmake/QGpgme/,libqgpgme.*} || true
+  rm -r "${pkgdir}"/usr/lib/python* || true
 }
 
 package_qgpgme() {
