commit 5bc30c1293e5e36b4cd6354193173a5ac55f12fa
Author: WANG Xuerui <git@xen0n.name>
Date:   Fri Jun 26 16:28:12 2020 +0800

    [stage2] libassuan: fix patch

diff --git a/PKGBUILD b/PKGBUILD
index c791ee3..c8eb10c 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -16,7 +16,7 @@ sha256sums=('91bcb0403866b4e7c4bc1cc52ed4c364a9b5414b3994f718c70303f7f765e702'
 
 build() {
 	cd "${srcdir}/${pkgname}-${pkgver}"
-	./configure --prefix=/usr
+	./configure --host=@CHOST@ --build=@BUILDHOST@ --prefix=/usr
 	make
 }
 
