diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2019-01-23 18:08:03.100897205 +0300
+++ b/PKGBUILD	2019-01-23 18:15:09.397148132 +0300
@@ -9,7 +9,7 @@ pkgdesc='POSIX 1003.1e capabilities'
 arch=('x86_64')
 url='http://sites.google.com/site/fullycapable/'
 license=('GPL2')
-depends=('glibc' 'attr')
+depends=('glibc' 'attr' 'pam' 'unixodbc' 'linux-api-headers')
 makedepends=('linux-api-headers')
 validpgpkeys=('38A644698C69787344E954CE29EE848AE2CCF3F4') # Andrew G. Morgan <morgan@kernel.org>
 source=(https://kernel.org/pub/linux/libs/security/linux-privs/libcap2/libcap-$pkgver.tar.{xz,sign})
@@ -26,7 +26,8 @@ prepare() {
 }
 
 build() {
-  make -C $pkgname-$pkgver KERNEL_HEADERS=/usr/include
+  make -C $pkgname-$pkgver KERNEL_HEADERS=@SYSROOT@/usr/include \
+    BUILD_CC=gcc BUILD_CFLAGS="" CC=@CHOST@-gcc AR=@CHOST@-ar RANLIB=@CHOST@-ranlib
 }
 
 package() {
