diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-09-11 23:43:33.000000000 +0300
+++ b/PKGBUILD	2019-02-05 15:55:39.903928119 +0300
@@ -30,12 +30,13 @@ pkgver() {
 
 prepare() {
   cd libtool
+  sed -i 's/depth 365/depth 1024/' ./bootstrap
   ./bootstrap
 }
 
 build() {
   cd libtool
-  ./configure --prefix=/usr lt_cv_sys_lib_dlsearch_path_spec="/usr/lib /usr/lib32"
+  ./configure --host=@CHOST@ --build=@BUILDHOST@ --prefix=/usr lt_cv_sys_lib_dlsearch_path_spec="/usr/lib /usr/lib32"
   make
 }
 
