commit 55f40e464b0fb10d207bc0faf9b2519d1add9991
Author: WANG Xuerui <git@xen0n.name>
Date:   Fri Jun 26 13:15:35 2020 +0800

    [stage2] libtool: fix patch

diff --git a/PKGBUILD b/PKGBUILD
index 10b6445..12005c7 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -44,12 +44,13 @@ prepare() {
   git config --local submodule.gl-mod/bootstrap.url "$srcdir/gnulib-bootstrap"
   git submodule update
 
+  sed -i 's/depth 365/depth 1024/' ./bootstrap
   ./bootstrap
 }
 
 build() {
   cd libtool
-  ./configure --prefix=/usr lt_cv_sys_lib_dlsearch_path_spec="/usr/lib /usr/lib32"
+  ./configure --host=@CHOST@ --build=@BUILDHOST@ --prefix=/usr lt_cv_sys_lib_dlsearch_path_spec="/usr/lib /usr/lib32"
   make
 }
 
