diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-09-11 21:53:17.000000000 +0300
+++ b/PKGBUILD	2019-02-05 11:02:48.184010994 +0300
@@ -20,12 +20,12 @@ build() {
   cd linux-$pkgver
 
   make mrproper
-  make headers_check
+  make ARCH=@LINUX_ARCH@ CROSS_COMPILE=@CHOST@- headers_check
 }
 
 package() {
   cd linux-$pkgver
-  make INSTALL_HDR_PATH="$pkgdir/usr" headers_install
+  make INSTALL_HDR_PATH="$pkgdir/usr" ARCH=@LINUX_ARCH@ CROSS_COMPILE=@CHOST@- headers_install
 
   # use headers from libdrm
   rm -r "$pkgdir/usr/include/drm"
