diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2019-01-23 17:55:51.062539349 +0300
+++ b/PKGBUILD	2019-01-23 18:02:21.757731758 +0300
@@ -31,7 +31,8 @@ build() {
 
 	# mark stack as non-executable: http://bugs.archlinux.org/task/12434
 	./Configure --prefix=/usr --openssldir=/etc/ssl --libdir=lib \
-		shared no-ssl3-method enable-ec_nistp_64_gcc_128 linux-x86_64 \
+		--cross-compile-prefix=@CHOST@- no-asm \
+		shared no-ssl3-method enable-ec_nistp_64_gcc_128 linux-elf \
 		"-Wa,--noexecstack ${CPPFLAGS} ${CFLAGS} ${LDFLAGS}"
 
 	make depend
