diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2019-01-24 15:46:12.000000000 +0300
+++ b/PKGBUILD	2019-02-05 16:26:27.029434148 +0300
@@ -33,10 +33,11 @@ prepare() {
 build() {
   cd p11-kit
   ./configure --prefix=/usr \
+    --host=@CHOST@ --build=@BUILDHOST@ \
     --sysconfdir=/etc \
     --localstatedir=/var \
     --libexecdir=/usr/lib \
-    --enable-doc \
+    --disable-doc \
     --with-module-path=/usr/lib/pkcs11 \
     --with-trust-paths=/etc/ca-certificates/trust-source:/usr/share/ca-certificates/trust-source
   make
