commit d05af9592dce1a379158d588763758bea95d7d12
Author: WANG Xuerui <git@xen0n.name>
Date:   Fri Jun 26 00:51:27 2020 +0800

    [stage2] pam: fix patch

diff --git a/PKGBUILD b/PKGBUILD
index 2fdbdcc..43e2952 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -8,7 +8,7 @@ pkgdesc="PAM (Pluggable Authentication Modules) library"
 arch=('x86_64')
 license=('GPL2')
 url="http://linux-pam.org"
-depends=('glibc' 'cracklib' 'libtirpc' 'pambase')
+depends=('glibc' 'cracklib' 'libtirpc' 'pambase' 'libnsl')
 makedepends=('flex' 'w3m' 'docbook-xml>=4.4' 'docbook-xsl')
 backup=(etc/security/{access.conf,group.conf,limits.conf,namespace.conf,namespace.init,pam_env.conf,time.conf} etc/default/passwd etc/environment)
 source=(https://github.com/linux-pam/linux-pam/releases/download/v$pkgver/Linux-PAM-$pkgver.tar.xz
@@ -24,7 +24,7 @@ options=('!emptydirs')
 
 build() {
   cd Linux-PAM-$pkgver
-  ./configure --libdir=/usr/lib --sbindir=/usr/bin --disable-db
+  ./configure --host=@CHOST@ --build=@BUILDHOST@ --libdir=/usr/lib --sbindir=/usr/bin --disable-db
   make
 }
 
@@ -37,5 +37,5 @@ package() {
 
   # remove doc which is not used anymore
   # FS #40749
-  rm "$pkgdir"/usr/share/doc/Linux-PAM/sag-pam_userdb.html
+  rm -f "$pkgdir"/usr/share/doc/Linux-PAM/sag-pam_userdb.html
 }
