commit bbb5e6cbb8b827840db4d42996d69deb1448c31d
Author: WANG Xuerui <git@xen0n.name>
Date:   Thu Jun 25 21:37:29 2020 +0800

    [stage2] perl: refresh pkgbuild

diff --git a/PKGBUILD b/PKGBUILD
index 1e560a6..f8f52bc 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -7,6 +7,7 @@
 pkgname=perl
 pkgver=5.32.0
 _baseversion="${pkgver%.*}"
+_crossversion=1.3.4
 pkgrel=1
 pkgdesc="A highly capable, feature-rich programming language"
 arch=(x86_64)
@@ -130,6 +131,7 @@ provides=('perl-archive-tar=2.36'
 # Add your own provides here
 provides=("${provides[@]}")
 source=(https://www.cpan.org/src/5.0/perl-${pkgver}.tar.xz
+        https://github.com/arsv/perl-cross/releases/download/${_crossversion}/perl-cross-${_crossversion}.tar.gz
         perlbin.sh
         perlbin.csh
         perlbin.fish
@@ -137,6 +139,7 @@ source=(https://www.cpan.org/src/5.0/perl-${pkgver}.tar.xz
         detect-old-perl-modules.hook)
 options=('makeflags' '!purge' 'emptydirs')
 sha512sums=('1540247415893bbd94dfeede7b4fba6052688dc0bf27ced817f448246fcdc6e9a6486abc34577dec5b00bf02ed607b2d24ccd4977c3b3c51e8e6edfc0b81c760'
+            '7e1fc7b2b19d8c2e2fb8615e074e218b2e1a4d276b563ffabccfd0c7398bc8680fca96da89cc61d611993f38a2c67dfbb0201c58658e05437395ad8c00d31874'
             'b7678078d64cc593a3503c45f023c49915d0d703f1cea8282f4191f1e3aa62764cc6cfcae3dc3828101415b8e15d5ed8b1b79d423ef387550ae33172b0de92bc'
             '53eb0cddfd637014f3d3a101665db8dcafe5ac5bf3d319a259974334eb89c1c405097518ae96b6d18e520194633c7be57c9b2cd9ae6398443eb08f1a2008d112'
             '881e2efe05ba818cd7300f126800b56bb0685cb5c9c5fb7e67ef6aaf5abd17d2391a979d5d16d109c5111f4b35504ba83d19b0e6eda4431e8421fcbea19d2f1a'
@@ -146,6 +149,7 @@ sha512sums=('1540247415893bbd94dfeede7b4fba6052688dc0bf27ced817f448246fcdc6e9a64
 
 prepare() {
   cd "${srcdir}/${pkgname}-${pkgver}"
+  tar --strip-components=1 -zxf ${srcdir}/perl-cross-${_crossversion}.tar.gz
 }
 
 build() {
@@ -159,7 +163,7 @@ build() {
     arch_opts=""
   fi
 
-  ./Configure -des -Dusethreads -Duseshrplib -Doptimize="${CFLAGS}" \
+  ./configure --target=@CHOST@ -Dusethreads -Duseshrplib -Doptimize="${CFLAGS}" \
     -Dprefix=/usr -Dvendorprefix=/usr \
     -Dprivlib=/usr/share/perl5/core_perl \
     -Darchlib=/usr/lib/perl5/$_baseversion/core_perl \
@@ -220,7 +224,7 @@ package() {
   install -d -m755 "$pkgdir/usr/bin/site_perl"
 
   #(cd ${pkgdir}/usr/bin; mv perl${pkgver} perl)
-  rm "$pkgdir/usr/bin/perl$pkgver"
+  rm -f "$pkgdir/usr/bin/perl$pkgver"
 
   install -D -m755 -t "$pkgdir/usr/share/libalpm/scripts" "$srcdir/detect-old-perl-modules.sh"
   install -D -m644 -t "$pkgdir/usr/share/libalpm/hooks" "$srcdir/detect-old-perl-modules.hook"
