diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2018-12-07 12:48:09.000000000 +0300
+++ b/PKGBUILD	2019-02-05 13:40:42.140431951 +0300
@@ -7,6 +7,7 @@
 pkgname=perl
 pkgver=5.28.1
 _baseversion="${pkgver%.*}"
+_crossversion=1.2.1
 pkgrel=1
 pkgdesc="A highly capable, feature-rich programming language"
 arch=(x86_64)
@@ -133,6 +134,7 @@ provides=('perl-archive-tar=2.30'
 # Add your own provides here
 provides=(${provides[@]})
 source=(https://www.cpan.org/src/5.0/perl-${pkgver}.tar.xz
+        https://github.com/arsv/perl-cross/releases/download/${_crossversion}/perl-cross-${_crossversion}.tar.gz
         perlbin.sh
         perlbin.csh
         perlbin.fish
@@ -140,6 +142,7 @@ source=(https://www.cpan.org/src/5.0/per
         detect-old-perl-modules.hook)
 options=('makeflags' '!purge' 'emptydirs')
 sha512sums=('6d18e9684c3a15bea2ccd28f116d1829c3acd5547551ee3539f0060c0d1a75246dfe570dfb9d5f00625a994a0afb0cbd6a5a5f9a407fef75a421e7dbc6491b43'
+            '26ea586e442f71cefe1e46a22fb8d03683df00a4e958b75394170ccebbf862ca09108fc822364bfcaa1c486fe073640aa2b6a078f8fd54685196203d90edbce5'
             '46724344828e7f86e016f9c8d588bf52b2e764e65e0acc1a38899a530c99bc6e4fd8b46fa0d4bbd685aa2074dd5bcbf9029ac3bb3f2d0ee9adfc4f6c0745f373'
             'fc1344a02c741d61af6f6b5967f29cc6f43c2059761522b150261924dd7e1989da5254c03ffa0627accd9af01bc152edd24e84a6b59579acb9ee1900b6ce9383'
             '881e2efe05ba818cd7300f126800b56bb0685cb5c9c5fb7e67ef6aaf5abd17d2391a979d5d16d109c5111f4b35504ba83d19b0e6eda4431e8421fcbea19d2f1a'
@@ -153,6 +156,7 @@ prepare() {
   # test broken with gdbm 1.15. See: https://rt.perl.org/Public/Bug/Display.html?id=133295
   sed -i 's|BEGIN {|BEGIN { plan(skip_all => "fatal test unsupported with gdbm 1.15");|' ext/GDBM_File/t/fatal.t
 
+  tar --strip-components=1 -zxf ${srcdir}/perl-cross-${_crossversion}.tar.gz
 }
 
 build() {
@@ -166,7 +170,7 @@ build() {
     arch_opts=""
   fi
 
-  ./Configure -des -Dusethreads -Duseshrplib -Doptimize="${CFLAGS}" \
+  ./configure --target=@CHOST@ -Dusethreads -Duseshrplib -Doptimize="${CFLAGS}" \
     -Dprefix=/usr -Dvendorprefix=/usr \
     -Dprivlib=/usr/share/perl5/core_perl \
     -Darchlib=/usr/lib/perl5/$_baseversion/core_perl \
@@ -180,7 +184,8 @@ build() {
     -Dinc_version_list=none \
     -Dman1ext=1perl -Dman3ext=3perl ${arch_opts} \
     -Dlddlflags="-shared ${LDFLAGS}" -Dldflags="${LDFLAGS}"
-  make
+  MAKEFLAGS=""
+  make miniperl; make
 }
 
 check() {
@@ -224,7 +229,7 @@ package() {
   install -d -m755 "$pkgdir/usr/bin/site_perl"
 
   #(cd ${pkgdir}/usr/bin; mv perl${pkgver} perl)
-  rm "$pkgdir/usr/bin/perl$pkgver"
+  rm -f "$pkgdir/usr/bin/perl$pkgver"
 
   install -D -m755 -t "$pkgdir/usr/share/libalpm/scripts" "$srcdir/detect-old-perl-modules.sh"
   install -D -m644 -t "$pkgdir/usr/share/libalpm/hooks" "$srcdir/detect-old-perl-modules.hook"
