diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2019-01-23 23:45:23.499402739 +0300
+++ b/PKGBUILD	2019-01-23 23:50:26.750586391 +0300
@@ -9,7 +9,7 @@ pkgdesc='Collection of simple PIN or pas
 url='http://gnupg.org/related_software/pinentry/'
 arch=('x86_64')
 license=('GPL')
-depends=('ncurses' 'libcap' 'libassuan' 'libsecret')
+depends=('ncurses' 'libcap' 'libassuan')
 makedepends=('gtk2' 'qt5-base' 'gcr')
 optdepends=('gtk2: gtk2 backend'
             'qt5-base: qt backend'
@@ -26,15 +26,17 @@ backup=('usr/bin/pinentry')
 build() {
 	cd "${srcdir}/${pkgname}-${pkgver}"
 	./configure \
+		--host=@CHOST@ --build=@BUILDHOST@ \
 		--prefix=/usr \
 		--enable-pinentry-tty \
 		--enable-pinentry-curses \
 		--enable-fallback-curses \
-		--enable-pinentry-emacs \
-		--enable-pinentry-gtk2 \
-		--enable-pinentry-gnome3 \
-		--enable-pinentry-qt \
-		--enable-libsecret \
+		--disable-pinentry-emacs \
+		--disable-pinentry-gtk2 \
+		--disable-pinentry-gnome3 \
+		--disable-pinentry-qt \
+		--disable-libsecret \
+		--disable-pinentry-fltk \
 
 	make
 }
