diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2019-01-23 18:16:06.761523611 +0300
+++ b/PKGBUILD	2019-01-23 18:17:00.599374851 +0300
@@ -50,6 +50,8 @@ pkgver() {
 prepare() {
   cd "$pkgname"
 
+  sed -i "/SUBDIRS/s/man//g" Makefile.am
+
   local backports=(
     # Fix usermod crash
     73a876a05612c278da747faeaeea40c3b8d34a53
@@ -74,12 +76,13 @@ build() {
 
   ./configure \
     LIBS="-lcrypt" \
+    --host=@CHOST@ --build=@BUILDHOST@ \
     --prefix=/usr \
     --bindir=/usr/bin \
     --sbindir=/usr/bin \
     --libdir=/usr/lib \
     --mandir=/usr/share/man \
-    --enable-man \
+    --disable-man \
     --sysconfdir=/etc \
     --with-libpam \
     --with-group-name-max-length=32 \
@@ -133,20 +136,6 @@ package() {
   mv "$pkgdir"/usr/bin/{newgrp,sg}
 
   # ...and their many man pages
-  find "$pkgdir"/usr/share/man \
-      '(' -name 'chsh.1'    -o \
-          -name 'chfn.1'    -o \
-          -name 'su.1'      -o \
-          -name 'logoutd.8' -o \
-          -name 'login.1'   -o \
-          -name 'nologin.8' -o \
-          -name 'vipw.8'    -o \
-          -name 'vigr.8'    -o \
-          -name 'newgrp.1' ')' \
-      -delete
-  rmdir \
-      "$pkgdir"/usr/share/man/{fi,id,zh_TW}/man1 \
-      "$pkgdir"/usr/share/man/{fi,ko/man8}
 
   # move everything else to /usr/bin, because this isn't handled by ./configure
   mv "$pkgdir"/usr/sbin/* "$pkgdir"/usr/bin
