commit 4548e304353c31b1cf70f6c8c14b6b363f72af0f
Author: WANG Xuerui <git@xen0n.name>
Date:   Fri Jun 26 11:25:57 2020 +0800

    [stage2] sqlite: fix patch

diff --git a/PKGBUILD b/PKGBUILD
index 34a38a4..8ecfeb7 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -12,6 +12,7 @@ pkgdesc="A C library that implements an SQL database engine"
 arch=('x86_64')
 license=('custom:Public Domain')
 url="https://www.sqlite.org/"
+depends=('tcl')
 makedepends=('tcl' 'readline' 'zlib')
 source=(https://www.sqlite.org/2020/sqlite-src-${_srcver}.zip
         https://www.sqlite.org/2020/sqlite-doc-${_docver}.zip
@@ -37,7 +38,7 @@ build() {
 
   # build sqlite
   cd sqlite-src-$_srcver
-  ./configure --prefix=/usr \
+  ./configure --host=@CHOST@ --build=@BUILDHOST@ --prefix=/usr \
 	--disable-static \
 	--disable-amalgamation \
 	--enable-fts3 \
@@ -46,6 +47,7 @@ build() {
 	--enable-rtree \
 	--enable-json1 \
 	TCLLIBDIR=/usr/lib/sqlite$pkgver
+  sed -i '/+= -I\/usr\/include/d' Makefile
   make
   # build additional tools
   make showdb showjournal showstat4 showwal sqldiff sqlite3_analyzer
