commit d1ff806c0e0dc60c64f91b43c27dfff3641aeb67
Author: WANG Xuerui <git@xen0n.name>
Date:   Fri Jun 26 16:48:59 2020 +0800

    [stage2] sudo: fix patch

diff --git a/PKGBUILD b/PKGBUILD
index 322ae12..2da9d60 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -11,7 +11,7 @@ arch=('x86_64')
 url="https://www.sudo.ws/sudo/"
 license=('custom')
 groups=('base-devel')
-depends=('glibc' 'libgcrypt' 'pam' 'libldap')
+depends=('glibc' 'libgcrypt' 'pam')
 backup=('etc/pam.d/sudo'
         'etc/sudo.conf'
         'etc/sudo_logsrvd.conf'
@@ -34,6 +34,7 @@ build() {
   cd "$srcdir/$pkgname-$_sudover"
 
   ./configure \
+    --host=@CHOST@ --build=@BUILDHOST@ \
     --prefix=/usr \
     --sbindir=/usr/bin \
     --libexecdir=/usr/lib \
@@ -44,8 +45,7 @@ build() {
     --enable-tmpfiles.d \
     --with-pam \
     --with-sssd \
-    --with-ldap \
-    --with-ldap-conf-file=/etc/openldap/ldap.conf \
+    --without-ldap \
     --with-env-editor \
     --with-passprompt="[sudo] password for %p: " \
     --with-all-insults
