commit bfe6b86a4bc7e2a40f6fbb426d3b52ab9bbc4da3
Author: WANG Xuerui <git@xen0n.name>
Date:   Thu Jun 25 23:32:42 2020 +0800

    [stage2] systemd: fix patch

diff --git a/PKGBUILD b/PKGBUILD
index 1016c45..e8eec45 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -13,9 +13,9 @@ makedepends=('acl' 'cryptsetup' 'docbook-xsl' 'gperf' 'lz4' 'xz' 'pam' 'libelf'
              'intltool' 'iptables' 'kmod' 'libcap' 'libidn2' 'libgcrypt'
              'libmicrohttpd' 'libxslt' 'util-linux' 'linux-api-headers'
              'python-lxml' 'quota-tools' 'shadow' 'gnu-efi-libs' 'git'
-             'meson' 'libseccomp' 'pcre2' 'audit' 'kexec-tools' 'libxkbcommon'
+             'meson' 'libseccomp' 'pcre2' 'kexec-tools' 'libxkbcommon'
              'bash-completion' 'p11-kit' 'systemd')
-options=('strip')
+depends=('util-linux' 'libutil-linux' 'pcre2' 'libidn' 'libseccomp' 'kmod')
 validpgpkeys=('63CDA1E5D3FC22B998D20DD6327F26951A015CC4'  # Lennart Poettering <lennart@poettering.net>
               '5C251B5FC54EB2F80F407AAAC54CA336CFEB557E') # Zbigniew Jędrzejewski-Szmek <zbyszek@in.waw.pl>
 source=("git+https://github.com/systemd/systemd-stable#tag=${_tag}?signed"
@@ -118,6 +118,20 @@ prepare() {
 }
 
 build() {
+  cat > cross-file << EOF
+[binaries]
+c = '@CHOST@-gcc'
+cpp = '@CHOST@-g++'
+ar = '@CHOST@-ar'
+strip = '@CHOST@-strip'
+pkgconfig = 'pkg-config'
+[host_machine]
+system = 'linux'
+cpu_family = '@CARCH@'
+cpu = '@GCC_MARCH@'
+endian = 'little'
+EOF
+
   local _timeservers=({0..3}.arch.pool.ntp.org)
   local _nameservers=(
     # We use these public name services, ordered by their
@@ -136,11 +150,17 @@ build() {
   local _meson_options=(
     -Dversion-tag="${pkgver}-${pkgrel}-arch"
 
-    -Dgnu-efi=true
+    -Dgnu-efi=false
     -Dima=false
     -Dlibidn2=true
     -Dlz4=true
-    -Dman=true
+    -Dman=false
+    -Dlibcryptsetup=false
+    -Dlibiptc=false
+    -Dxkbcommon=false
+    -Dlibcurl=false
+    -Dtests=false
+    -Daudit=false
 
     # We disable DNSSEC by default, it still causes trouble:
     # https://github.com/systemd/systemd/issues/10579
@@ -159,7 +179,9 @@ build() {
     -Dsysvrcnd-path=
   )
 
-  arch-meson "$pkgbase-stable" build "${_meson_options[@]}"
+  export PKG_CONFIG_LIBDIR="@SYSROOT@/usr/lib/pkgconfig"
+  export PKG_CONFIG_SYSROOT_DIR="@SYSROOT@"
+  arch-meson "$pkgbase-stable" build "${_meson_options[@]}" --cross-file cross-file
 
   ninja -C build
 }
@@ -175,7 +197,7 @@ package_systemd() {
            'iptables' 'kbd' 'kmod' 'libkmod.so' 'hwids' 'libcap' 'libcap.so'
            'libgcrypt' 'systemd-libs' 'libidn2' 'libidn2.so' 'lz4' 'pam'
            'libelf' 'libseccomp' 'libseccomp.so' 'util-linux' 'libblkid.so'
-           'libmount.so' 'xz' 'pcre2' 'audit' 'libaudit.so' 'libp11-kit'
+           'libmount.so' 'xz' 'pcre2' 'libp11-kit'
            'libp11-kit.so')
   provides=('nss-myhostname' "systemd-tools=$pkgver" "udev=$pkgver")
   replaces=('nss-myhostname' 'systemd-tools' 'udev')
@@ -203,20 +225,20 @@ package_systemd() {
   DESTDIR="$pkgdir" meson install -C build
 
   # we'll create this on installation
-  rmdir "$pkgdir"/var/log/journal/remote
+  rm -rf "$pkgdir"/var/log/journal/remote
 
   # runtime libraries shipped with systemd-libs
   install -d -m0755 systemd-libs
   mv "$pkgdir"/usr/lib/lib{nss,systemd,udev}*.so* systemd-libs
 
   # manpages shipped with systemd-sysvcompat
-  rm "$pkgdir"/usr/share/man/man8/{halt,poweroff,reboot,runlevel,shutdown,telinit}.8
+  rm -f "$pkgdir"/usr/share/man/man8/{halt,poweroff,reboot,runlevel,shutdown,telinit}.8
 
   # executable (symlinks) shipped with systemd-sysvcompat
   rm "$pkgdir"/usr/bin/{halt,init,poweroff,reboot,runlevel,shutdown,telinit}
 
   # files shipped with systemd-resolvconf
-  rm "$pkgdir"/usr/{bin/resolvconf,share/man/man1/resolvconf.1}
+  rm -f "$pkgdir"/usr/{bin/resolvconf,share/man/man1/resolvconf.1}
 
   # avoid a potential conflict with [core]/filesystem
   rm "$pkgdir"/usr/share/factory/etc/{issue,nsswitch.conf}
@@ -277,8 +299,8 @@ package_systemd-resolvconf() {
   install -d -m0755 "$pkgdir"/usr/bin
   ln -s resolvectl "$pkgdir"/usr/bin/resolvconf
 
-  install -d -m0755 "$pkgdir"/usr/share/man/man1
-  ln -s resolvectl.1.gz "$pkgdir"/usr/share/man/man1/resolvconf.1.gz
+  #install -d -m0755 "$pkgdir"/usr/share/man/man1
+  #ln -s resolvectl.1.gz "$pkgdir"/usr/share/man/man1/resolvconf.1.gz
 }
 
 package_systemd-sysvcompat() {
@@ -287,8 +309,8 @@ package_systemd-sysvcompat() {
   conflicts=('sysvinit')
   depends=('systemd')
 
-  install -D -m0644 -t "$pkgdir"/usr/share/man/man8 \
-    build/man/{telinit,halt,reboot,poweroff,runlevel,shutdown}.8
+  #install -D -m0644 -t "$pkgdir"/usr/share/man/man8 \
+  #  build/man/{telinit,halt,reboot,poweroff,runlevel,shutdown}.8
 
   install -d -m0755 "$pkgdir"/usr/bin
   ln -s ../lib/systemd/systemd "$pkgdir"/usr/bin/init
