commit ec101ba9699a535483dad2cef81563580acb219b
Author: WANG Xuerui <git@xen0n.name>
Date:   Fri Jun 26 01:30:17 2020 +0800

    [stage2] util-linux: fix patch

diff --git a/PKGBUILD b/PKGBUILD
index 1ec605a..efd8b33 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -41,6 +41,8 @@ build() {
   # https://github.com/karelzak/util-linux/issues/808
 
   ./configure \
+    --host=@CHOST@ --build=@BUILDHOST@ \
+    --without-systemd \
     --prefix=/usr \
     --libdir=/usr/lib \
     --bindir=/usr/bin \
@@ -55,7 +57,7 @@ build() {
     --enable-write \
     --enable-mesg \
     --disable-hardlink \
-    --with-python=3
+    --without-python
 
   make
 }
@@ -64,7 +66,7 @@ package_util-linux() {
   conflicts=('rfkill')
   provides=('rfkill')
   replaces=('rfkill')
-  depends=('pam' 'shadow' 'coreutils' 'systemd-libs' 'libcap-ng' 'libutil-linux')
+  depends=('pam' 'shadow' 'coreutils' 'libcap-ng' 'libutil-linux')
   optdepends=('python: python bindings to libmount'
               'words: default dictionary for look')
   backup=(etc/pam.d/chfn
@@ -92,7 +94,7 @@ package_util-linux() {
   install -m644 "$srcdir/pam-su" "$pkgdir/etc/pam.d/su-l"
 
   # TODO(dreisner): offer this upstream?
-  sed -i '/ListenStream/ aRuntimeDirectory=uuidd' "$pkgdir/usr/lib/systemd/system/uuidd.socket"
+  #sed -i '/ListenStream/ aRuntimeDirectory=uuidd' "$pkgdir/usr/lib/systemd/system/uuidd.socket"
 
   # adjust for usrmove
   # TODO(dreisner): fix configure.ac upstream so that this isn't needed
@@ -118,6 +120,7 @@ package_util-linux() {
 
 package_libutil-linux() {
   pkgdesc="util-linux runtime libraries"
+  depends=(pam shadow coreutils libcap-ng)
   provides=('libblkid.so' 'libfdisk.so' 'libmount.so' 'libsmartcols.so' 'libuuid.so')
 
   make -C "$pkgbase-$pkgver" DESTDIR="$pkgdir" install-usrlib_execLTLIBRARIES
