diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2019-01-29 01:28:04.406090964 +0300
+++ b/PKGBUILD	2019-01-29 01:37:25.770910696 +0300
@@ -12,7 +12,7 @@ url="https://www.kernel.org/pub/linux/ut
 arch=('x86_64')
 makedepends=('systemd' 'python' 'libcap-ng')
 license=('GPL2')
-options=('strip' 'debug')
+options=('debug')
 validpgpkeys=('B0C64D14301CC6EFAEDF60E4E4B71D5EEC39C284')  # Karel Zak
 source=("https://www.kernel.org/pub/linux/utils/util-linux/v$_pkgmajor/$pkgbase-$pkgver.tar."{xz,sign}
         pam-{login,common,runuser,su}
@@ -35,6 +35,8 @@ build() {
   cd "$pkgbase-$pkgver"
 
   ./configure \
+    --host=@CHOST@ --build=@BUILDHOST@ \
+    --without-systemd \
     --prefix=/usr \
     --libdir=/usr/lib \
     --bindir=/usr/bin \
@@ -46,7 +48,7 @@ build() {
     --enable-chfn-chsh \
     --enable-write \
     --enable-mesg \
-    --with-python=3
+    --without-python
 
   make
 }
@@ -55,7 +57,7 @@ package_util-linux() {
   conflicts=('rfkill')
   provides=('rfkill')
   replaces=('rfkill')
-  depends=('pam' 'shadow' 'coreutils' 'libsystemd' 'libcap-ng' 'libutil-linux')
+  depends=('pam' 'shadow' 'coreutils' 'libcap-ng' 'libutil-linux')
   optdepends=('python: python bindings to libmount'
               'words: default dictionary for look')
   groups=('base' 'base-devel')
@@ -84,7 +86,6 @@ package_util-linux() {
   install -m644 "$srcdir/pam-su" "$pkgdir/etc/pam.d/su-l"
 
   # TODO(dreisner): offer this upstream?
-  sed -i '/ListenStream/ aRuntimeDirectory=uuidd' "$pkgdir/usr/lib/systemd/system/uuidd.socket"
 
   # adjust for usrmove
   # TODO(dreisner): fix configure.ac upstream so that this isn't needed
@@ -110,6 +111,7 @@ package_util-linux() {
 
 package_libutil-linux() {
   pkgdesc="util-linux runtime libraries"
+  depends=(pam shadow coreutils libcap-ng)
   provides=('libblkid.so' 'libfdisk.so' 'libmount.so' 'libsmartcols.so' 'libuuid.so')
 
   make -C "$pkgbase-$pkgver" DESTDIR="$pkgdir" install-usrlib_execLTLIBRARIES
