diff -rupN a/PKGBUILD b/PKGBUILD
--- a/PKGBUILD	2019-01-24 01:10:35.708641455 +0300
+++ b/PKGBUILD	2019-01-24 01:13:52.464587057 +0300
@@ -9,18 +9,23 @@ arch=('x86_64')
 license=('LGPL2.1')
 url="https://github.com/seccomp/libseccomp"
 depends=('glibc')
-validpgpkeys=('7100AADFAE6E6E940D2E0AD655E45A5AE8CA7C8A') # Paul Moore <paul@paul-moore.com>
-source=("https://github.com/seccomp/$pkgname/releases/download/v$pkgver/$pkgname-$pkgver.tar.gz"{,.asc})
-sha256sums=('7fc28f4294cc72e61c529bedf97e705c3acf9c479a8f1a3028d4cd2ca9f3b155'
-            'SKIP')
+makedepends=('git')
+source=("git+https://github.com/Icenowy/libseccomp.git#branch=riscv")
+sha256sums=('SKIP')
+
+pkgver() {
+  echo "2.3.3"
+}
 
 build() {
-  cd $pkgname-$pkgver
-  ./configure --prefix=/usr
+  cd $pkgname
+  autoreconf --install
+  sed -i 's/(EM_RISCV64|/(EM_RISCV|/' tools/util.c
+  ./configure --host=@CHOST@ --build=@BUILDHOST@ --prefix=/usr
   make
 }
 
 package() {
-  cd $pkgname-$pkgver
+  cd $pkgname
   make DESTDIR="$pkgdir" install
 }
